IaC/pg.cd/JenkinsStack.yml

```diff
@@ -613,6 +613,11 @@ Resources:
                   echo JENKINS_HOME=/mnt/$JENKINS_HOST \
                       | tee -a /etc/sysconfig/jenkins
                   sed -i 's^JENKINS_ARGS=""^JENKINS_ARGS="--sessionTimeout=10080 --sessionEviction=86400"^' /etc/sysconfig/jenkins
+                  JAVA_HOME=$(alternatives --display java | grep current | sed 's/link currently points to //' | sed 's|/bin/java||' | xargs)
+                  echo "$JAVA_HOME" > /etc/profile.d/java.sh
+                  echo JENKINS_JAVA_CMD=$JAVA_HOME/bin/java \
+                      | tee -a /etc/sysconfig/jenkins
```

Comment:
can we add JAVA_HOME also into `/etc/sysconfig/jenkins` ?

Reply:
Nope, we can't
But the JENKINS_JAVA_CMD we can